source/solvers/gls_sharp_navier_stokes.cc

```diff
@@ -150,7 +151,7 @@ GLSSharpNavierStokesSolver<dim>::point_inside_cell(
         this->mapping->transform_real_to_unit_cell(cell, point);
       const double dist = GeometryInfo<dim>::distance_to_unit_cell(p_cell);
       // if the cell contains the point, the distance is equal to 0
-      if (dist == 0)
+      if (dist <= 1e-12)
```

Comment:
Can you change that so that the tolerance is a multiple of the cell size? Like 1e-6 * cell_size or something?

Reply:
it's on the unite cell so it's always one